packages/cli/src/cli/commands/auth/login.ts

```diff
@@ -0,0 +1,79 @@
+import chalk from 'chalk';
+import { createCommand } from 'commander';
+import { ConsoleLogger } from '../../../bin/utils/ConsoleLogger.js';
+import { initializeFramework } from '../../../lib/framework.node.js';
+
+import { withAuthOptions } from '../../options/auth.js';
+
+/**
+ * Command to authenticate and log in to Fusion Framework using interactive browser-based authentication.
+ *
+ * This command initializes the Fusion Framework and opens a browser window for secure login.
+ * Supports custom tenant, client, and scope options for advanced authentication scenarios.
+ *
+ * @example
+ *   $ fusion login
+ *   $ fusion login --tenant my-tenant --client my-client-id --scope api://my-app/.default
+ */
+const DEFAULT_SERVER_PORT = 49741 as const;
```

Comment:
Shoulde this be documented somewhere. Seems realy importatn the this dont change in the future. and if it is changed then it will nee to be changed ont tha app reg as well :)